receiver/gitproviderreceiver/README.md

```diff
@@ -9,15 +9,18 @@
 | [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@adrielp](https://www.github.com/adrielp), [@andrzej-stencel](https://www.github.com/andrzej-stencel) |
 
 [development]: https://github.com/open-telemetry/opentelemetry-collector#development
-<!-- end autogenerated section --> 
+<!-- end autogenerated section -->
 
-The Git Provider receiver scrapes data from Git vendors.
+The Git Provider receiver scrapes data from Version Control Systems (VCS) that use Git as their underlying technology.
```

Comment:
@crobert-1  - my only pause on this pull request is this statement right here. I want to move this to alpha and get it into contrib right after this pull request. However, my pause is at the name of the receiver. I have no plans for supporting mercurial, or SVN. However, that doesn't meant the community won't in the future. Plus, all the calls are API based, so it's not like `git` actually matters under the hood. 

Thoughts on renaming this receiver from `gitproviderreceiver` to `vcsproviderreceiver`?

Reply:
Discussed async, will rename as separate pull request. I have no intentions of contributing code to support systems like mercurial, etc, however if the community wants it, they're welcome to. Despite that intention, this still more closely aligns with SemConv v1.27.0 (which is getting ready to be released).